### PR TITLE
prepare 1.17 ✈️

### PR DESCRIPTION
**Pull request by r10s**
this is for a testflight release ✈️

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Delta Chat iOS Changelog
 
+## 1.17.0
+2021-03
+
+* new mailinglist and better bot support
+* support reliable notifications at least every 20 minutes
+* add option to view original-/html-mails
+* allow dialing on tapping a phone number
+* support more audio formats
```
Comment (cyBerta) [inline CHANGELOG.md:10]: is this done by the core? 
What we did UI-wise was only to show errors if a audio format is not supported.

**Comment by cyBerta** [comment]
Besides my comment, this is a quite impressive list :) +1

File: CHANGELOG.md (hunk 1 above)
Comment (r10s) [reply CHANGELOG.md:10]: ah, i mixed that up, thanks for the hint.

**Comment by r10s** [comment]
merging that in, however for a testflight, the go-to-background after 30 seconds after didReceiveRemoteNotification is missing; that may to notifications being throttled (noticed that after checking my device this morning), i created an issue for that https://github.com/deltachat/deltachat-ios/issues/1099

**Pull request merged**